src/views/Dashboard/Customer/Home/Phr/Edit.vue

```diff
@@ -353,20 +356,20 @@ export default {
       try {
         this.isDocumentLoading = true
         const { id } = this.$route.params
-        const data = await queryElectronicMedicalRecordById(this.api, id)
+        const emerData = await queryElectronicMedicalRecordById(this.api, id)
         let files = []
 
-        if (!id || !data) {
+        if (!id || !emerData) {
           this.messageError = "Oh no! We can't find your selected order. Please select another one or try again"
 
           return
         }
 
-        this.emr.id = id
-        this.emr.title = data.title
-        this.emr.category = data.category
+        this.phr.id = id
+        this.phr.title = emerData.title
+        this.phr.category = emerData.category
```

Comment:
can you check the grammar, do you mean `emrData` ? you can change to emrData or phrData